src/sql/parse.py

```diff
@@ -10,6 +10,17 @@
 
 from sql import exceptions
 
+SQL_COMMANDS = [
```

Comment:
add a comment here to explain why we need this

Reply:
Added explanation [here](https://github.com/ploomber/jupysql/pull/938/files#diff-22cfc6ec3e879941494b36e0dffaf24c100ab50173c413d821ebab94d0627ddfR13-R16)